build/build-image/Dockerfile

```diff
@@ -25,7 +25,8 @@ RUN chmod -R a+rwx /usr/local/go/pkg
 # of operations.
 ENV HOME /go/src/k8s.io/kubernetes
 WORKDIR ${HOME}
-RUN chmod -R a+rwx ${HOME}
+RUN mkdir -p ${HOME} ${HOME}/_output \
```

Comment:
Why do we need to created and chmod _output directory?

(For chmod you are doing it recursively it anyway, so you don't need it, right?)